fineract-provider/src/main/resources/application.properties

```diff
@@ -164,3 +164,10 @@ fineract.activemq.standingInstructionInsufficientBalanceFailureQueue=${INSUFFICI
 fineract.configuration.resetNuban=${CONFIGURATION_RESET_NUBAN:false}
 fineract.configuration.nubanCode=${CONFIGURATION_NUBAN_CODE:00000}
 fineract.configuration.pentahoFolderName=${PENTAHO_FOLDER_NAME:pentahoReports}
+
+//odoo integration
```

Comment:
@Deepika1095  can you provide a default value to avoid fineract making noise for tenants who don't need ODOO

Reply:
![image](https://github.com/fiterafrica/fineract-template/assets/57247520/8b7fcba3-64a3-4490-abea-1f01e0249440)
you mean this type